src/main/java/hudson/plugins/git/extensions/impl/DisableAutomaticScheduling.java

```diff
@@ -0,0 +1,30 @@
+package hudson.plugins.git.extensions.impl;
+
+import hudson.Extension;
+import hudson.plugins.git.extensions.GitSCMExtension;
+import hudson.plugins.git.extensions.GitSCMExtensionDescriptor;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+/**
+ * Don't trigger another build to catch up
+ *
+ * @author Sven Hickstein
+ */
+public class DisableAutomaticScheduling extends GitSCMExtension {
+    @DataBoundConstructor
+    public DisableAutomaticScheduling() {
+    }
+
+    @Override
+    public boolean disableAutomaticScheduling() {
+        return true;
+    }
+
+    @Extension
+    public static class DescriptorImpl extends GitSCMExtensionDescriptor {
+        @Override
+        public String getDisplayName() {
+            return "Disable \"Triggered by an SCM change\"";
```

Comment:
I don't think that the user will understand "Triggered by an SCM change" as the action they are disabling when they add this extension.

I'm not much in favor of phrasal verbs, but other areas in the code seem to call it a "catch up" merge or something similar.  Can you find other words to describe the capability you're allowing the user to disable?

Specifically, the user needs to understand that if multiple changes are detected which match the branch specification, only one of those changes will be built.  The other changes will not be scheduled to be built.